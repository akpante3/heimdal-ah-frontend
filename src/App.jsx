/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import Routes from './routes/routes';
import 'font-awesome/css/font-awesome.min.css';

class App extends Component {
  render() {
    return (
      <Router>
        <Routes />
      </Router>
    );
  }
}

export default App;
