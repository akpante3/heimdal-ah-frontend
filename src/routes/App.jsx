/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';

import LoginPage from '../component/Pages/LoginPage.jsx';
import Homepage from '../component/Pages/HomePage.jsx';

class App extends Component {
  render() {
    return (
      <Router>
        <div className='container'>
          <Switch>
            <Route exact path='/' component={Homepage} />
          </Switch>
          <Switch>
            <Route exact path='/login' component={LoginPage} />
          </Switch>
        </div>
      </Router>
    );
  }
}

export default App;
