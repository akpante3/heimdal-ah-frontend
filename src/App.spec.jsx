/* eslint-disable no-undef */
import React from 'react';
import toJson from 'enzyme-to-json';
import App from './App';

describe('Testing the App', () => {
  it('should render the Routes', () => {
    const wrapper = shallow(<App />);
    const routes = wrapper.find('Routes');
    expect(routes.length).toBe(1);
  });

  it('should match the snapshot', () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
