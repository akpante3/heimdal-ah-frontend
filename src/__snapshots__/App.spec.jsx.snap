// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the App should match the snapshot 1`] = `
<BrowserRouter>
  <Routes />
</BrowserRouter>
`;
